src/Test/PdbUtilities/Pdb/SymReaderFactory.cs

```diff
@@ -23,7 +23,7 @@ public static class SymReaderFactory
         [DllImport("Microsoft.DiaSymReader.Native.amd64.dll", EntryPoint = "CreateSymReader")]
         private extern static void CreateSymReader64(ref Guid id, [MarshalAs(UnmanagedType.IUnknown)]out object symReader);
 
-        private static ISymUnmanagedReader3 CreateNativeSymReader(Stream pdbStream, object metadataImporter)
+        private static ISymUnmanagedReader5 CreateNativeSymReader(Stream pdbStream, object metadataImporter)
```

Comment:
The compiler doesn't use ISymUnmanagedReader, only ISymUnmanagedWriter. This helper only loads SymReader from DSRN where it always is the latest version.